lib/tasks/bulk_import_documents_from_csv.rake

```diff
@@ -88,7 +88,7 @@ def get_hearing_officer_mapping
 end
 
 def get_design_decision_british_library_number(title)
-  title[/Design hearing decision:\s*(\S+)/i, 1] if title
+  title[/([O0]\/\d+\/\d+)/i, 1] if title
```

Comment:
Fix up to commit where this is introduced (and add appropriate test so that if we reverted this change, something fails)